src/middlewares/ReadingStatMiddleware.js

```diff
@@ -0,0 +1,36 @@
+import models from '../db/models';
+import Response from '../helpers/response';
+
+const { Art } = models;
+
+/** Reading Stat Class */
+class ReadingStatMiddleware {
+  /**
+   * @static
+   *
+   * @param {object} req
+   * @param {object} res
+   * @param {object} next
+   * @returns {*} - object containing the message, status and code
+   * @memberof ReadingStatMiddleware
+   */
+  static async getStat(req, res, next) {
+    try {
+      const read = await Art
```

Comment:
Add an attribute field to specify the exact field you need from the query.

Reply:
fixed